lib/release-ops.js

```diff
@@ -128,7 +144,7 @@ function getVersionTags() {
  * @private
  */
 function parseLogs(logs) {
-    const regexp = /^\* ([0-9a-f]{40}) ((?:([a-z]+): ?)?.*) \((.*)\)/i,
+    const regexp = /^\* ([0-9a-f]{40}) ((?:([a-z]+!?): ?)?.*) \((.*)\)/i,
```

Comment:
I've noticed some contributors include a scope as part of their conventional commits-style commit message, e.g. `fix(no-console): ...`. Do we want to allow scopes?

Reply:
I’d prefer not to, at least to start. I’m not convinced scopes are all that useful outside of a monorepo.